crates/sui/src/keytool.rs

```diff
@@ -69,9 +67,12 @@ pub enum KeyToolCommand {
         /// The alias must start with a letter and can contain only letters, digits, hyphens (-), or underscores (_).
         new_alias: Option<String>,
     },
-    /// Convert private key from wallet format (hex of 32 byte private key) to sui.keystore format
```

Comment:
Just a note that we're changing/overloading the semantics of this command, which folks used to convert a private key from the wallet to the sui.keystore format. We should mention to use `keytool import `instead if wanting to import a key from the wallet, if it's already using a version that's newer than x.x.x (the version which will no longer support hex).